css/index.css

```diff
@@ -1,3 +1,436 @@
-/* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body p {
+	line-height: 1.2;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+  height: 100%;
+  font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+  font-family: 'Bangers', cursive;
+  letter-spacing: 1px;
+  margin-bottom: 15px;
+}
+
+.container {
+width: 880px;
+/* TB LR */
+margin: 0 auto;
+}
+
+/* header */
+
+header {
+display: flex;
+flex-wrap: column;
+}
+header nav {
+display: flex;
+width: 691px;
+vertical-align: top;
+margin-top: 35px;
+}
+
+header nav a {
+margin-right: 65px;
+font-size: 16px;
+font-weight: bold;
+color: gray;
+text-decoration: none;
+}
+
+header nav a:hover {
+text-decoration: underline;
+}
+
+header nav a:first-child {
+margin-left: 0;
+}
+
+header nav a:last-child {
+margin-right: 0;
+}
+
+header img.logo {
+margin-top: 25px;
+}
+
+/* cta */
+.cta {
+  display: flex;
+}
+
+.cta .cta-text {
+vertical-align: top;
+font-size: 77px;
+margin-top: 120px;
+text-align: center;
+padding: 0px 75px;
+}
+
+.cta .cta-text button {
+background: white;
+vertical-align: top;
+margin-top: 6px;
+padding: 8px 44px;
+font-size: 17px;
+border: 1px solid black;
+cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+background: black;
+color: white;
+}
+
+.cta img {
+margin-top: 69px;
+align-self: flex-start;
+}
+
+.main-content {
+border-top: 2px solid black;
+margin-top: 76px;
+padding-top: 39px;
+}
+
+.main-content .top-content {
+display: flex;
+padding: 0 10px;
+justify-content: space-between
+}
+
+.main-content .top-content .text-content {
+width: 410px;
+margin-right: 34px;
+}
+
+.main-content .top-content .text-content:last-child {
+margin-right: 0;
+}
+
+.main-content .middle-img {
+  margin-top: 45px;
+}
+
+.main-content .bottom-content {
+margin-top: 37px;
+padding: 0 10px;
```

Comment:
Try using prettier (https://prettier.io/) for indenting your code uniformly so that you don't have to do it by hand!